lib/data/model/cats_model.dart

```diff
@@ -0,0 +1,8 @@
+import 'dart:typed_data';
+
+class CatsImageFactModel {
+  CatsImageFactModel({this.fact, this.image});
```

Comment:
made const contructor
add my linter rules https://gist.github.com/zddarova/577850ad9125c3753ae8784e0c1df28d and fix all warnings